pivotal-tracker/pivotal_integration.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+// Get the environment; we will post a new comment to Tracker each time a commit appears on Pantheon.
+$env = $_ENV['PANTHEON_ENVIRONMENT'];
+
+// Do not watch test or live, though.
```

Comment:
Hm, what is the actual strategy that should be used here? e.g. if you mark an issue as 'fixed' in a commit in a multidev environment, and then you delete the multidev without ever merging it with the dev environment, then Pivotal Tracker will say that the issue is fixed, but the commit will be lost. Maybe this should only operate when `env` is `dev`?